src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -41,6 +43,34 @@ export default function CSVFileImport({ url, title }: CSVFileImportProps) {
     // });
     // console.log("Result: ", result);
     // setFile("");
+    localStorage.setItem(
+      "authorization_token",
+      "Wm9raXJQcm89VEVTVF9QQVNTV09SRA=="
+    );
```

Comment:
It looks like I can't check that you have a correct error processing for an incorrect or empty token because you set it just before the request.